Containerfile.task

```diff
@@ -7,4 +7,6 @@
 FROM quay.io/redhat-user-workloads/rhtap-build-tenant/buildah-container/buildah@sha256:7cb5a35b7fe44e397fbf3b834f3bd8dcd9403a7c0a0b51469e6ec75b107d0846
 RUN microdnf install -y rsync openssh-clients kubernetes-client && \
     microdnf -y clean all && \
-    rm -rf /var/cache /var/log/dnf* /var/log/yum.*
\ No newline at end of file
+    rm -rf /var/cache /var/log/dnf* /var/log/yum.*
+RUN curl -L https://github.com/keilerkonzept/dockerfile-json/releases/download/1.0.8/dockerfile-json_1.0.8_linux_x86_64.tar.gz | tar -xz -C /usr/bin
+RUN curl -L https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux64 -o /usr/bin/jq && chmod +x /usr/bin/jq
```

Comment:
Are these both arch-specific binaries? We build this image as multi-arch so we cannot simply curl a single binary for each image. This image is based on Fedora, so we can install `jq` from an RPM. You can include `dockerfile-json` as a git submodule an build the binary in an earlier stage if no other option is available.

Reply:
Updated to build from source